InvenTree/templates/js/build.js

```diff
@@ -686,14 +688,12 @@ function loadBuildOutputAllocationTable(buildInfo, output, options={}) {
                     return makeProgressBar(allocated, required);
                 },
                 sorter: function(valA, valB, rowA, rowB) {
+                    console.log("sorting!");
```

Comment:
@SchrodingersGat  is that there on purpose or from testing?

Reply:
Thanks @matmair was just a debug statement